build.sbt

```diff
@@ -10,7 +10,7 @@ libraryDependencies ++= Seq(
   ws,
   specs2 % Test,
   "com.gu" %% "pan-domain-auth-play_2-4-0" % "0.2.7",
-  "com.amazonaws" % "aws-java-sdk" % "1.7.5"
+  "com.amazonaws" % "aws-java-sdk" % "1.10.50"
```

Comment:
I like it better when you specify only what you actually need from aws instead of loading the entire sdk. But it's not a big deal, especially for apps. Different story for libs.

Why not upgrading panda as well as your making some changes?